node/src/accountant/db_access_objects/failed_payable_dao.rs

```diff
@@ -0,0 +1,778 @@
+// Copyright (c) 2025, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+use crate::accountant::db_access_objects::utils::{TxHash, TxIdentifiers};
+use crate::accountant::db_big_integer::big_int_divider::BigIntDivider;
+use crate::accountant::{checked_conversion, comma_joined_stringifiable};
+use crate::database::rusqlite_wrappers::ConnectionWrapper;
+use masq_lib::utils::ExpectValue;
+use std::collections::{HashMap, HashSet};
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+use web3::types::Address;
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum FailedPayableDaoError {
+    EmptyInput,
+    NoChange,
+    InvalidInput(String),
+    PartialExecution(String),
+    SqlExecutionFailed(String),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum FailureReason {
+    PendingTooLong,
+    NonceIssue,
```

Comment:
We could extend this in the future, right? If we begin to read the topics from the tx receipts...